bootstrap/cmd/kfctl/cmd/delete.go

```diff
@@ -42,11 +44,15 @@ var deleteCmd = &cobra.Command{
 		if err != nil {
 			return fmt.Errorf("cannot fetch current directory for apply: %v", err)
 		}
-		deleteStorage := deleteCfg.GetBool(string(kftypes.DELETE_STORAGE))
-		kfApp, err = coordinator.GetKfAppFromCfgFile(cwd+"/app.yaml", deleteStorage)
+		kfApp, err = coordinator.BuildKfAppFromURI(filepath.Join(cwd, "app.yaml"))
```

Comment:
This will attempt to recreate the app.yaml file - why is this being changed?

Reply:
Discussed with @jlewi , this PR needs to be rebased after https://github.com/kubeflow/kubeflow/pull/4275 is merged.

Reply:
With #4275  BuildKFAppFromURI should no longer be attempting to recreate the KFDef file. A new config file should only be created in the event we are using a remote URI. But this isn't a remote URI so we should be good.